packages/client/.eslintrc.js

```diff
@@ -1,25 +1,16 @@
 module.exports = {
   extends: [
-    "plugin:@typescript-eslint/eslint-recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:react/recommended",
+    "react-app",
     "plugin:prettier/recommended",
-    "prettier/@typescript-eslint"
   ],
-  plugins: ["@typescript-eslint", "prettier", "react"],
+  plugins: [],
   env: { node: true, es6: true },
   parser: "@typescript-eslint/parser",
   parserOptions: {
     sourceType: "module",
     project: "./tsconfig.json",
   },
-  rules: {
-    // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
-    // e.g. "@typescript-eslint/explicit-function-return-type": "off",
-    "react/jsx-props-no-spreading": "off",
-    "no-use-before-define": "off",
-    "@typescript-eslint/no-use-before-define": "off"
-  },
+  rules: {},
   settings:  {
     react:  {
       version:  'detect',  // Tells eslint-plugin-react to automatically detect the version of React to use
```

Comment:
I think we can remove these as they are sort of included in the base eslint config and I am not sure if it will get overridden. 🤷‍♂️

Reply:
Removed!